packer/ansible/roles/localsource/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+---
+- name: install dev libraries for NPM installs
+  apt: pkg={{ item }} state=installed
+  with_items:
+    - libkrb5-dev
+    - unzip
+  sudo: yes
+
+- name: Install RackHD from local source
+  copy: src="../../{{ item }}" dest="{{ ansible_env.HOME }}/src/"
+  with_items:
+    - on-core
+    - on-tasks
+    - on-http
+    - on-tftp
+    - on-dhcp-proxy
+    - on-taskgraph
+    - on-syslog
+    - on-wss
+    - on-tools
+    - on-imagebuilder
```

Comment:
This list of repos duplicates what is downloaded in the remoterepos playbook:
https://github.com/RackHD/RackHD/blob/master/packer/ansible/roles/remoterepos/tasks/main.yml